components/size_aware_image.jsx

```diff
@@ -148,10 +147,22 @@ export default class SizeAwareImage extends React.PureComponent {
                         />
                     </div>
                 }
-                <img
-                    {...props}
-                    src={src}
-                />
+                {dimensions && dimensions.width && !this.state.loaded ? (
+                    <div className={`image-loading__container ${this.props.className}`}>
+                        <div>
```

Comment:
Why do we need this inner div? It seems unnecessary

Reply:
Yup. Not necessary removed it.